sparse_strips/vello_hybrid/examples/text.rs

```diff
@@ -0,0 +1,260 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Draws some text on screen.
+
+mod common;
+
+use common::{RenderContext, RenderSurface, create_vello_renderer, create_winit_window};
+use parley::FontFamily;
+use parley::{
+    Alignment, AlignmentOptions, FontContext, GlyphRun, Layout, LayoutContext,
+    PositionedLayoutItem, StyleProperty,
+};
+use skrifa::instance::NormalizedCoord;
+use std::sync::Arc;
+use vello_common::color::palette::css::WHITE;
+use vello_common::color::{AlphaColor, Srgb};
+use vello_common::glyph::Glyph;
+use vello_hybrid::{RenderParams, Renderer, Scene};
+use wgpu::RenderPassDescriptor;
+use winit::{
+    application::ApplicationHandler,
+    event::WindowEvent,
+    event_loop::{ActiveEventLoop, EventLoop},
+    window::{Window, WindowId},
+};
+
+const ROBOTO_FONT: &[u8] = include_bytes!("../../../examples/assets/roboto/Roboto-Regular.ttf");
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+struct ColorBrush {
+    color: AlphaColor<Srgb>,
+}
+
+impl Default for ColorBrush {
+    fn default() -> Self {
+        Self { color: WHITE }
+    }
+}
+
+fn main() {
+    let mut app = App {
+        context: RenderContext::new(),
+        font_cx: FontContext::new(),
+        layout_cx: LayoutContext::new(),
+        renderers: vec![],
+        state: RenderState::Suspended(None),
+        scene: Scene::new(900, 600),
+    };
+
+    app.font_cx.collection.register_fonts(ROBOTO_FONT.to_vec());
```

Comment:
One of "always using roboto" and collecting system fonts is unnecessary.

In terms of teaching how we expect people to use this, I'd probably lean towards only using system fonts (acknowledging the difficulty that has with web support - but this example doesn't yet run on the web)

Reply:
Use system fonts in https://github.com/linebender/vello/pull/883/commits/9ab73412723bd9336585ba84700c101528aeefad and https://github.com/linebender/vello/pull/883/commits/d4d5658cb7151e7f4a7e580484ca2f3401a52c0c

Reply:
Ah, it fails CI when we try to build the `vello_hybrid` crate to Wasm. Instead of opting `vello_hybrid` out of Wasm build, I've reverted these changes. I will implement them as part of this [idea](https://github.com/linebender/vello/pull/883#discussion_r2023744690) next.